### DISCUSSION TITLE
Select Page Location: All Folders and Pages are Shown Twice

### DISCUSSION THREAD

**Discussion by Marco77577** [requarks/wiki#3728] [Error / Bug Report]
**Describe the bug**
When creating or moving a page, the "Select Page Location" dialog displays all folders and pages twice since to upgrade from 2.0.1 to 2.1.113.

Folders are even shown in the wrong locations (child folders are displayed like top-level folders).

**To Reproduce**
1. Create page or move page.
2. View the folders and pages.

**Expected behavior**
Every folder and page is only shown once.

**Screenshots**
![Capture](https://user-images.githubusercontent.com/36480635/74735472-0cdf2280-5251-11ea-809c-9621b3ec7238.PNG)
The folders in the green rectangle are actually located within /technical, but are shown as if they were top level folders. When clicking one of them, /technical opens and the correct folder is selected.

All pages to the right are shown twice. This happens for all the folders. When selecting one, both are selected, suggesting that they do indeed not exist twice (I checked the database, only one page per path).

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: 2.1.113
 - Database engine: postgres 11.6

**Comment by kingfisher77** [comment]
Same here.

OS: CentOS 8.1
Wiki.js version: 2.1.113
PostgreSQL 12.2

**Comment by JackThursby252** [comment]
We are in the process of changing our internal documentation system, so I spun up multiple test instances of wikijs lately ("full" installation and docker). Noticed this in every installation we made so far.... Seems like a nasty bug, since sometimes the pages are even shown 4 times, even though i don't know how I triggered that. 

/Edit: Only tried Wiki.JS 2.1.113 with postgreSQL 11 and 12.

**Comment by SFdmpg** [comment]
+1

**Comment by m4n1du** [comment]
2.2.51 seems to have fixed the "duplicate page" issue. However, folders are still pulled out of the hierarchy and are shown as top level.

**Comment by annahri** [comment]
Latest update fixed the page duplicates, yet I'm still seeing folder duplicates.

**Comment by Marco77577** [comment]
@annahri Are you sure they are duplicates, or are they just "pulled out of the hierarchy" to the top level, as manraf mentioned above?

**Comment by SFdmpg** [comment]
1+ to what @manraf  said

**Comment by m4n1du** [comment]
> 
> 
> 2.2.51 seems to have fixed the "duplicate page" issue. However, folders are still pulled out of the hierarchy and are shown as top level.

Just to clarify - the pages appear **both** in the correct hierarchy as well as on the first level.

**Comment by annahri** [comment]
> @annahri Are you sure they are duplicates, or are they just "pulled out of the hierarchy" to the top level, as manraf mentioned above?

Ah yes, that's what I meant.
